packages/base-controller/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- **BREAKING:** Change the types for `BaseControllerV1` class fields `initialConfig`, `initialState` from `C`, `S` to `Partial<C>`, `Partial<S>` ([#3959](https://github.com/MetaMask/core/pull/3959))
+- **BREAKING:** Convert `BaseConfig`, `BaseState` interfaces to types ([#3959](https://github.com/MetaMask/core/pull/3959))
```

Comment:
Since we're aiming to remove BaseControllerV1, maybe we don't worry about this for now? This way we can avoid a breaking change, at least for this PR.

Reply:
Reverted here: https://github.com/MetaMask/core/pull/3959/commits/8347a2acb748fb291c84176dcbe10b9b44b0261f